extern/rust/Makefile.mk

```diff
@@ -6,13 +6,13 @@ LIBCORE_DIR=$(BUILD_DIR)/core-$(RUSTC_VERSION)
 
 $(EXTERN_DIR)rust/rustc-$(RUSTC_VERSION)-src.tar.gz:
 	@echo "Need libcore to compile Tock: fetching source $(@F)"
-	@wget -q -O $@ https://github.com/rust-lang/rust/archive/$(RUSTC_VERSION).tar.gz
+	@wget -q -O $@ https://github.com/rust-lang/rust/archive/$(RUSTC_VERSION).tar.gz --no-check-certificate
 
 $(EXTERN_DIR)rust/rustc/src/libcore/lib.rs: $(EXTERN_DIR)rust/rustc-$(RUSTC_VERSION)-src.tar.gz
 	@echo "Extracting $(<F)"
 	@rm -rf $(EXTERN_DIR)/rust/rustc
 	@mkdir -p $(EXTERN_DIR)/rust/rustc
-	@tar -C $(EXTERN_DIR)/rust/rustc -zx --strip-components=1 -f $^
+	@tar -C $(EXTERN_DIR)/rust/rustc -zx --strip-components=1 -f $^ --force-local
```

Comment:
I guess this is a problem because on Windows `EXTERN_DIR` expands to something that starts with, e.g. `C:\\...`?